src/components/Header/Header.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { BellIcon, SettingsIcon } from '@chakra-ui/icons';
+import {
+  Flex,
+  Heading,
+  IconButton,
+  Text,
+  useColorModeValue
+} from '@chakra-ui/react';
+import i18next from 'i18next';
+
+import { LogoWalletIcon, UserIcon } from '@/assets';
+import theme from '@/theme';
+
+import ColorModeToggle from '../ColorModeToggle';
+
+import Navbar from './Navbar';
+
+const Header = () => {
+  const [isAuth, setIsAuth] = useState(false);
+
+  const headerBgThemeColor = useColorModeValue('#D6BCFA', '#553C9A');
+  const iconsThemeColor = useColorModeValue('#322659', '#fff');
```

Comment:
use chakras color constants instead of your own colors 
<img width="508" alt="image" src="https://user-images.githubusercontent.com/52533333/178958362-7e4d2473-b84a-4728-a3da-97c950fd2673.png">

see the whole list of chakra colors here https://chakra-ui.com/docs/styled-system/theme

Reply:
done